mozci/queries/push_tasks_from_unittest.query

```diff
@@ -2,7 +2,6 @@ from: unittest
 select:
     - {name: id, value: task.id}
     - {name: kind, value: task.kind}
-    - {name: _groups, value: run.suite.groups}
```

Comment:
Was this intentional?

Reply:
@ahal  That was a mistake.  Although, that is big field (an array of all groups), which works out a a few gigabytes per query.